l_bsp_q3.c

```diff
@@ -692,14 +798,14 @@ void	Q3_WriteBSPFile( char *filename )
 	Q3_AddLump( bspfile, header, Q3_LUMP_SURFACES, q3_drawSurfaces, q3_numDrawSurfaces*sizeof(q3_dsurface_t) );
 	Q3_AddLump( bspfile, header, Q3_LUMP_VISIBILITY, q3_visBytes, q3_numVisBytes );
 	Q3_AddLump( bspfile, header, Q3_LUMP_LIGHTMAPS, q3_lightBytes, q3_numLightBytes );
-	Q3_AddLump( bspfile, header, Q3_LUMP_LIGHTGRID, q3_gridData, 8 * q3_numGridPoints );
+	Q3_AddLump( bspfile, header, Q3_LUMP_LIGHTGRID, q3_gridData, q3_numGridBytes );
```

Comment:
This change confused me for a bit. I think it's fine but I would rather it had stayed consistent with the q3map2 implementation that counts in grid points, especially since bspc doesn't seem to do anything with this.